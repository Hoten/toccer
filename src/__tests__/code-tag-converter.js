import codeTagConverter from '../code-tag-converter'
import toMarkdown from 'to-markdown'

describe('codeTagConverter', function () {
  const htmlWithoutLang = `<pre><code>console.log(1)</code></pre>`
  const htmlWithLang = `<pre><code class="lang-js">console.log(1)</code></pre>`

  it('works without lang', function () {
    expect(toMarkdown(htmlWithoutLang, {converters: [codeTagConverter]})).toMatchSnapshot()
  })

  it('works with lang', function () {
    expect(toMarkdown(htmlWithLang, {converters: [codeTagConverter]})).toMatchSnapshot()
  })
})
