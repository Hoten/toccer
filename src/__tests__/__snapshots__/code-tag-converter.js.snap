// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`codeTagConverter works with lang 1`] = `
"\`\`\`js
console.log(1)
\`\`\`"
`;

exports[`codeTagConverter works without lang 1`] = `
"\`\`\`
console.log(1)
\`\`\`"
`;
